packages/@glimmer/runtime/lib/vm/attributes/dynamic.ts

```diff
@@ -215,3 +235,20 @@ function normalizeValue(value: Opaque): Option<string> {
 
   return String(value);
 }
+
+function normalizeBooleanValue(value: Opaque): Option<boolean> {
```

Comment:
Based on https://github.com/glimmerjs/glimmer-vm/pull/841/files#diff-3ce638612baaf18381842996696f1f37R219, but for `boolean` values.